docs/statistics.md

```diff
@@ -705,3 +718,86 @@ The value is used to apply TSBPD delay for reading the received data on the sock
+| [pktRecvUniqueTotal](#group-pktRecvUniqueTotal) | accumulated       | packets             | -                    | ✓                      | int64_t   |
+| [pktRcvDropTotal](#group-pktRcvDropTotal)       | accumulated       | packets             | -                    | ✓                      | int32_t   |
+| [pktRcvDiscardTotal](#group-pktRcvDiscardTotal) | accumulated       | packets             | -                    | ✓                      | int32_t   |
+|                                                 |                   |                     |                      |                        |           |
+|                                                 |                   |                     |                      |                        |           |
+|                                                 |                   |                     |                      |                        |           |
+|                                                 |                   |                     |                      |                        |           |
+
+## Accumulated Statistics <a name="group-accumulated-statistics"></a>
+
+### msTimeStamp <a name="group-msTimeStamp"></a>
+
+The time elapsed, in milliseconds, since the time ("connection" time) when the initial group connection has been initiated (the time when the first connection in the group has been made and therefore made the group connected). This "connection" time will be then set in this statistic in every next socket that will become a member of the group as the new connections are established. A new connection to an already connected group doesn’t change the value of "connection" time. Available both for sender and receiver. 
+
+### pktSentUniqueTotal <a name="group-pktSentUniqueTotal"></a>
+
+The number of *unique original* DATA packets sent by the socket group. Available for sender.
+
+This value counts every *original* DATA packet sent over the network for the first time by the socket group. There is no difference between Connection Bonding modes (broadcast, backup and balancing). For example, sending the packet with a particular sequence number over multiple links in case of broadcast mode (it means sending this packet multiple times) does not affect the statistic and this very packet is taken into account only once.
+
+This statistic does not count retransmitted DATA packets that are individual per socket connection within the group. See the corresponding [pktRetransTotal](#pktRetransTotal) socket statistic.
+
+If the `SRTO_PACKETFILTER` socket option is enabled (refer to [API.md](https://cac-word-edit.officeapps.live.com/we/API.md)), this statistic does not count packet filter control packets that are individual per socket connection within the group. See the corresponding [pktSndFilterExtraTotal](#pktSndFilterExtraTotal) socket statistic.
+
+### pktRecvUniqueTotal <a name="group-pktRecvUniqueTotal"></a>
```

Comment:
I think if you simply change the title to "Group pktRecvUniqueTotal" it should have the same effect.